prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="file:///Users/steveportillo/repos/prep-0721-code-solutions/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>
```

Comment:
Rather than use the absolute file path to the `contact-us.html`, use a relative path to the `contact-us.html` inside of the nearby `contact-us/` directory (_i.e._ `href="contact-us/contact-us.html"`).

That way, the link works on computers other than your own.